### DISCUSSION TITLE
Having a way to remove the 6px frame but don't know if you want it

### DISCUSSION THREAD

**Discussion by littlewhitecloud** [Zingzy/hPyT#6] [Ideas]
Before:
![image](https://github.com/Zingzy/hPyT/assets/71159641/c44275b5-8a69-4e44-adc4-9f671c81d470)
After:
![image](https://github.com/Zingzy/hPyT/assets/71159641/5bcf2e43-19d2-48fd-9046-55f2c2240aff)

**Comment by littlewhitecloud** [comment]
It also works with each type of window.
Code like this:
```python
def handle(hwnd: int, msg: int, wp: int, lp: int) -> int:
        if msg == WM_NCCALCSIZE and wp:
            lpncsp = NCCALCSIZE_PARAMS.from_address(lp)
            lpncsp.rgrc[0].top -= 6

        return windll.user32.CallWindowProcW(*map(c_uint64, (globals()[old], hwnd, msg, wp, lp)))

old, new = "old_wndproc", "new_wndproc"
prototype = WINFUNCTYPE(c_uint64, c_uint64, c_uint64, c_uint64, c_uint64)

globals()[old] = None
globals()[new] = prototype(handle)
globals()[old] = windll.user32.GetWindowLongPtrA(self.hwnd, GWL_WNDPROC)
windll.user32.SetWindowLongPtrA(HWND, GWL_WNDPROC, globals()[new])
```

**Reply by Zingzy** [reply]
Hello, I really appreciate your suggestion for removing the 6px frame that appears after using the `titlebar.hide` function. However, I tried to implement your code and I encountered some errors and issues. Could you please be more elaborate on how your code works? Alternatively, you could open a pull request with the suggested changes, and I would be happy to review it and test it.

**Reply by littlewhitecloud** [reply]
Thanks! I may take time to test it. I used this code in one of my repo: https://github.com/littlewhitecloud/CustomTkinterTitlebar
If you want to learn more about it, you can view it.

**Comment by Zingzy** [comment]
Issue Fixed by @littlewhitecloud, the feature would be live in the next release
